skip-tests.json

```diff
@@ -178,6 +178,7 @@
     },
     "bitfinex1": {
         "skipMethods": {
+            "features": "not implemented for this old ex",
```

Comment:
I dont have implemented `features` it for `bitfinex1`, as it's obsolete and to be removed soon